src/main/java/io/reactivex/Flowable.java

```diff
@@ -13391,6 +13391,8 @@ public final void subscribe(FlowableSubscriber<? super T> s) {
     /**
      * Returns a Flowable that emits those items emitted by source Publisher before a specified time runs
      * out.
+     * If time runs out before Flowable is finished, termination events would be balled on provided {@link Scheduler},
```

Comment:
A `<p>` would cleanly separate the first sentence and the rest of the Javadoc. Also the wording could use some refinement:

```
<p>
If time runs out before the {@code Flowable} completes normally, the {@code onComplete} 
event will be signaled on the default {@code computation} {@link Scheduler}.
```